docs/en/2-Publishing/Datasette.md

```diff
@@ -0,0 +1,82 @@
+# Explore your data with Datasette
+
+Datasette is an instant JSON API for your SQLite databases allowing you to
+explore the DB and run SQL queries in a more interactive way.
+
+You can find a list of example datasettes
+[here](https://github.com/simonw/datasette/wiki/Datasettes).
+
+<!-- prettier-ignore -->
+!!! faq "The Datasette Ecosystem"
+    There are all sorts of tools for converting data to and from sqlite 
+    [here](https://docs.datasette.io/en/stable/ecosystem.html). For example,
+    you can load shapefiles into sqlite, or create [Vega](https://vega.github.io/vega/)
+    plots from a sqlite database. SQLite works well with `R`, `Python`, and many other tools.
+
+## Example Datasette
+
+Screenshots from
+[global-power-plants](https://global-power-plants.datasettes.com), you can
+preview and explore the data in the browser, either with clicks or SQL queries.
+
+![Preview Data](../images/datasette-preview.png)
+
+You can even explore maps within the tool!
+
+![Run SQL Queries](../images/datasette-sql.png)
+
+# Starting Datasette
```

Comment:
I think this is missing part of the example?  It references an `app.py` but I don't see where that gets created.

Reply:
Good catch. Fixed.